rn/Teacher/package.json

```diff
@@ -54,7 +54,6 @@
     "react-native-button": "^2.1.0",
     "react-native-camera": "^3.23.1",
     "react-native-counter": "Kerumen/react-native-counter",
-    "react-native-device-info": "^0.10.1",
```

Comment:
We weren't even ever using the webview?

Reply:
This package used UIWebView to get the user agent for the device. At one point we must have been using this package, but apparently we haven't been for a while.